manifests/plugin/rabbitmq.pp

```diff
@@ -102,7 +102,7 @@
 
   exec { 'update_typesdb':
     command => "/bin/cat ${rt} >> ${ct}",
-    unless  => "[ `cat ${ct} | sort -u | wc -l` -eq `cat ${rt} ${ct} | sort -u | wc -l` ] || exit 1",
+    unless  => "true && case $(cat ${ct}) in *$(cat ${rt})*) exit 0;; *) exit 1;; esac",
```

Comment:
@Yuav Hack looks hacky.  Did you try `provider => shell`? https://docs.puppet.com/puppet/latest/types/exec.html#exec-provider-shell